### PR TITLE
doc(readme): create value proposition

### PR DESCRIPTION
**Pull request by jdrouet**
Signed-off-by: Jérémie Drouet <jeremie.drouet@gmail.com>

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -34,3 +34,63 @@ with the following dependencies
...
+    },
+    {
+      "name": "tool-c",
+      "path": "common/cmd/tool-c"
+    }
+  ]
+}
+```
+
+The project will create the checksum of each project, check which project changed and then resolve what has be built.
```
Comment (mat007) [inline readme.md]: `has be` -> `has to be`

File: readme.md [hunk 2]
```diff
@@ -34,3 +34,63 @@ with the following dependencies
...
+For example, changing `tool-a` would output the following
```
Comment (mat007) [inline readme.md]: Maybe using a change on `tool-b` would be better, highlighting that change detection is transitive e.g. `tool-a` will need to be rebuilt, then the two apps as well?

**Review by mat007 — COMMENTED** [review]
I'm wondering if there isn't a tool doing that already (component dependency management or something like that), have you looked around a bit?

**Comment by jdrouet** [comment]
@mat007 I didn't search deeply. But it sounds like a makefile that does the diff with a reference.

File: readme.md (hunk 1 above)
Comment (jdrouet) [reply readme.md]: Also check if the project changed : how and compared to what. This has to be specified in the project file. Like "compare this to master, with a checksum, ignore the .md files"

**Pull request closed without merging**